src/utils/list.js

```diff
@@ -1,42 +1,60 @@
-const getListStyleType = (listType) => {
-  switch (listType) {
-    case 'upper-roman':
-      return 'upperRoman';
-    case 'lower-roman':
-      return 'lowerRoman';
-    case 'upper-alpha':
-    case 'upper-alpha-bracket-end':
-      return 'upperLetter';
-    case 'lower-alpha':
-    case 'lower-alpha-bracket-end':
-      return 'lowerLetter';
-    case 'decimal':
-    case 'decimal-bracket':
-    default:
-      return 'decimal';
+class ListStyleBuilder {
+  // defaults is an object passed in from constants.js / numbering with the following properties:
+  // defaultOrderedListStyleType: 'decimal' (unless otherwise specified)
+  constructor(defaults) {
+    if (!defaults) {
+      throw new Error(
+        'ListStyleBuilder requires a defaults object. See constants.js / numbering for an example.'
+      );
+    }
+
+    this.defaults = defaults;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  getListStyleType(listType) {
+    switch (listType) {
+      case 'upper-roman':
+        return 'upperRoman';
+      case 'lower-roman':
+        return 'lowerRoman';
+      case 'upper-alpha':
+      case 'upper-alpha-bracket-end':
+        return 'upperLetter';
+      case 'lower-alpha':
+      case 'lower-alpha-bracket-end':
+        return 'lowerLetter';
+      case 'decimal':
+      case 'decimal-bracket':
+      default:
+        return 'decimal';
+    }
   }
-};
 
-const getListPrefixSuffix = (style, lvl) => {
-  if (!style) return `%${lvl + 1}`;
+  getListPrefixSuffix(style, lvl) {
+    let listType = this.defaults.defaultOrderedListStyleType;
+
+    if (style && style['list-style-type']) {
+      listType = style['list-style-type'];
+    }
 
-  const listType = style['list-style-type'];
-  switch (listType) {
-    case 'upper-roman':
-    case 'lower-roman':
-    case 'upper-alpha':
-    case 'lower-alpha':
-      return `%${lvl + 1}.`;
-    case 'upper-alpha-bracket-end':
-    case 'lower-alpha-bracket-end':
-    case 'decimal-bracket-end':
-      return `%${lvl + 1})`;
-    case 'decimal-bracket':
-      return `(%${lvl + 1})`;
-    case 'decimal':
-    default:
-      return `%${lvl + 1}.`;
+    switch (listType) {
+      case 'upper-roman':
+      case 'lower-roman':
+      case 'upper-alpha':
+      case 'lower-alpha':
+        return `%${lvl + 1}.`;
+      case 'upper-alpha-bracket-end':
+      case 'lower-alpha-bracket-end':
+      case 'decimal-bracket-end':
+        return `%${lvl + 1})`;
+      case 'decimal-bracket':
+        return `(%${lvl + 1})`;
+      case 'decimal':
+      default:
+        return `%${lvl + 1}.`;
+    }
   }
-};
+}
 
-export { getListStyleType, getListPrefixSuffix };
+export { ListStyleBuilder as default };
```

Comment:
why not use default export since its probably the only thing to be exported from this file